lib/tasks/govuk_assets.rake

```diff
@@ -16,4 +16,27 @@ namespace :govuk_assets do
     status = result.successful? ? 'OK' : 'Error'
     puts "#{status}: #{result.written_count} documents updated"
   end
+
+  desc 'Unreplace an asset and delete the replacement'
+  task :unreplace_asset, [:uuid] => :environment do |_, args|
+    begin
+      uuid = args[:uuid]
+      asset = Asset.unscoped.find_by(uuid: uuid)
+
+      if asset.replacement.nil?
+        puts "#{uuid} has no replacement"
+        return
+      end
+
+      puts "found replacement #{asset.replacement.uuid}"
+
+      asset.replacement.deleted = true
+      asset.replacement.save
+
+      asset.replacement = nil
+      asset.save
+    rescue Mongoid::Errors::DocumentNotFound
```

Comment:
I wouldn't worry too much about catching this exception or showing nice error messages, I think if the Rake task fails, it's fine to show the full traceback because it will be being used by developers.